homeassistant/components/device_automation/action.py

```diff
@@ -37,6 +43,18 @@ async def async_call_action_from_config(
         """Execute a device action."""
         raise NotImplementedError
 
+    def async_get_action_capabilities(
+        self, hass: HomeAssistant, config: ConfigType
+    ) -> GetAutomationCapabilitiesResult | Awaitable[GetAutomationCapabilitiesResult]:
+        """List action capabilities."""
+        return {}
```

Comment:
I don't think we should return here, in `async_get_actions` below, or in `async_validate_action_config` and `async_call_action_from_config` above.

Does mypy complain if the returns are removed?